src/sentry/integrations/base.py

```diff
@@ -463,3 +468,72 @@ def notify_remove_external_team(self, external_team: ExternalActor, team: Team)
 
     def remove_notification_settings(self, actor_id: int, provider: str) -> None:
         pass
+
+
+def is_response_success(resp) -> bool:
+    if resp.status_code and resp.status_code < 300:
+        return True
+    return False
+
+
+def is_response_error(resp) -> bool:
+    if not resp.status_code:
+        return False
+    return resp.status_code >= 400 and resp.status_code != 429 and resp.status_code < 500
+
+
+def disable_integration(
```

Comment:
Is there any way you can use the [shared_integration version of this](https://github.com/getsentry/sentry/blob/master/src/sentry/shared_integrations/client/base.py#L418)? Like could that be moved somewhere everyone can access it? It feels bad to have the same code twice.

Reply:
? i moved it out of `shared_integation`, everywhere is importing from here now

Reply:
oh nvm I can't read